andela/hc-wits-kla
# #158175322 Set different priorities on checks.

#### What does this PR do?
* Have different priorities (High, Normal, Low) set on checks.
#### Description of Task to be completed?
* Enable setting of different priority levels on checks.
* Enable escalation of checks to team members
* Modify UI, so that a user can set different priorities on checks.
#### How should this be manually tested?
* Clone the project from GitHub ```git clone https://github.com/andela/hc-wits-kla.git``` and set it up as indicated in the `README.md` file.
* After cloning, cd into the project directory and run the environment setup in the project README
* Run `python manage.py makemigrations`.
* Start the project `python manage.py runserver`.
*  Login and add a new check. You should be able to set priority for the created check i.e. High, Normal or Low.
#### What are the relevant pivotal tracker stories?
[158175322](https://www.pivotaltracker.com/story/show/158175322)
#### Screenshots (if appropriate)
Screenshot before implementation
<img width="1440" alt="screen shot 2018-07-09 at 16 14 42" src="https://user-images.githubusercontent.com/39955305/42452762-b0208b48-8393-11e8-9f99-df8a130194dc.png">

Screenshot after the implementation
<img width="1213" alt="screen shot 2018-07-20 at 14 02 47" src="https://user-images.githubusercontent.com/39955305/42999251-af6581d2-8c25-11e8-8fb2-ecd9f38ddb14.png">


<img width="1195" alt="screen shot 2018-07-20 at 13 29 43" src="https://user-images.githubusercontent.com/39955305/42999153-58393b88-8c25-11e8-8962-16affee4a7b6.png">
